src/eterna/AnnotationManager.ts

```diff
@@ -1188,7 +1188,7 @@ export default class AnnotationManager {
                     base.x + pose.xOffset,
                     base.y + pose.yOffset
                 );
-                view.display.position = new Point(
```

Comment:
fun fact, a lot of the changes going to Pixi 5 were alterations in how points worked (a class I've forgotten about went away) so the fact that we're bringing in a new Point variation is, honestly, kind of nice. I bet it will be a stark improvement.